patchelf.1

```diff
@@ -79,6 +79,10 @@ the "/foo/lib" reference instead of the "/tmp" entry.
 .IP --print-rpath
 Prints the DT_RUNPATH or DT_RPATH for an executable or library.
 
+.IP --update-rpath
+Updates the DR_RPATH to a DT_RUNTPATH or vice versa if "--force-rpath" is given
```

Comment:
Can you elaborate on when you want to set the one over the other?

Reply:
They have slightly different semantics as detailed in `man ld-linux`. Would saying that + linking to the other manpage be ok? Or should I add a small summary?

Reply:
I think this get a different name. It's already confusing that we say --print-rpath and it's actually printing the DT_RUNPATH. "Updates" might be also the wrong verb as it convert from DR_RPATH to a DT_RUNPATH, whereas updates clashes already with `--set-rpath`